aws/rust-runtime/aws-http/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 repository = "https://github.com/awslabs/smithy-rs"
 
 [dependencies]
-aws-smithy-http = { path = "../../../rust-runtime/aws-smithy-http" }
+aws-smithy-http = { path = "../../../rust-runtime/aws-smithy-http", features = ["bytestream-util"] }
```

Comment:
Another question for @rcoh: If most of our AWS runtime crates pull in the `bytstream-util` feature, is there any point to it being a feature? Was the intent that Smithy generated clients/servers can opt out of it?

Reply:
I assumed that not all SDKs had streaming data and maybe that's why it exists?

Reply:
yeah, it's a feature because it uses `tokio/fs` and `tokio/io`

I'm not sure why anything other than end users would need it though